18F/identity-idp
# LG-9951: Add confirmation prompt when user selects backup code as sole MFA

## 🎫 Ticket

[LG-9951: Add confirmation prompt when user selects backup code as sole MFA](https://cm-jira.usa.gov/browse/LG-9951)

## 🛠 Summary of changes

This PR asks users to confirm that they have saved their backup codes before completing the signup process

## 📜 Testing Plan

On localhost:3000

- Follow the steps to create a new account
- Select backup codes as your setup method option
- Continue to receive backup codes
- Follow steps to set up backup codes
- On the "You’ve added your first authentication method!" screen, click "Skip for now"
- See the new confirmation page

## 👀 Screenshots

![image](https://github.com/18F/identity-idp/assets/17969963/a9a86eff-7459-4080-8aaf-e1f54e926953)
